setup.py

```diff
@@ -178,6 +178,16 @@ def _setup_entry_points() -> Dict:
         ]
     )
 
+    # object segmentation integration
+
+    entry_points["console_scripts"].extend(
+        [
+            "sparseml.yolact.export_onnx=sparseml.yolact.scripts:export",
```

Comment:
I do not like the inconsistency in the entry point naming. 

E.g.
`Image Classification` name is being used for all the entrypoints for IC across sparseml and deepsparse.

However,
`yolo` name is being used for entrypoints in sparseml, while in deepsparse `yolo` is called `object_detection`
I personally started calling yolact `instance_segmentation` (in deepsparse's entrypoints) to follow the convention, while you use `yolact` naming. Let's standardize that to avoid confusion.